app/helpers/users_helper.rb

```diff
@@ -0,0 +1,8 @@
+module UsersHelper
+  # Returns the Gravatar for the given user.
+  def gravatar_for(user)
+    gravatar_id = Digest::MD5.hexdigest(user.email.downcase)
+    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}"
+    image_tag(gravatar_url, alt: user.first_name, class: 'gravatar')
```

Comment:
is there a chance that user don't have one? some default?

Reply:
gravatar provides his own default avatar